cursor/cursor.go

```diff
@@ -200,8 +200,9 @@ func (m *Model) SetChar(char string) {
 
 // View displays the cursor.
 func (m Model) View() string {
-	if m.Blink {
-		return m.TextStyle.Render(m.char)
+	style := m.TextStyle.Inline(true)
```

Comment:
I don't think we really need to allocate a variable here (to be fair, I'd guess the Go compiler does optimize this for us), we can use the same return logic as before and add an `Inline(true)` to the style.